cns/internal/heartbeat.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2018 Microsoft. All rights reserved.
+// MIT License
+
+package internal
```

Comment:
why `internal` specifically? package should reflect what the code's domain/intention is. `internal` is used for structuring packages and disallow importing, but is general in the path of a package, not the name of the package itself. please think about a more descriptive package name.